src/css/global.css

```diff
@@ -0,0 +1,96 @@
+/* stylelint-disable docusaurus/copyright-header */
+/**
+* Any CSS included here will be global. The classic template
+* bundles Infima by default. Infima is a CSS framework designed to
+* work well for content-centric websites.
+*/
+/* You can override the default Infima variables here. */
```

Comment:
Was this CSS copied from somewhere? Please link to the original source, especially if the source requires attribution per license

Reply:
This part of the code came from the auto-generated .scss file from docusaurus